### PR TITLE
Adding basic building guidelines into readme

### LINKED ISSUE #3: Write a simple building guidelines
It can be written some guidelines for a simple build of a project with this library. It might take place in a section of the `README.md` file.

I have done a basic workflow to achieve a project build with such. Could I be assigned to this issue?

### PR DESCRIPTION
**Pull request by alfosua**
fixes: #3 

 *  Adds basic building guidelines for library
 *  Adds basic building guidelines for projects which use this library

Notes:

 * Command line specified was in Bash only. (I'm not aware if it could work in others like PowerShell, and so.)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,58 @@
...
+
+**Note: this command was only ran on Bash at the time. It may vary according to your command line.**
+
+**Note: release build type is not configured at the time yet.**
+
+If everything goes fine, you might have these files in your build directory: `./bin/test_cpprex` and `./lib/libcpprex_.a`. These, asides the ones generated by cmake and conan. If so, you can test the build by running `ctest`. (On sources from stable branches should pass all tests.)
+
+### Building a project with this library
+
+If you want to link this library to a project for usage, you may need to first build separately this project as the prior guidelines remarks. We recommend putting the library source code and binaries at `./deps/cpprex` relative directory of your application project source code directory. Make use of [Git submodules](https://gist.github.com/gitaarik/8735255) for integrate the source code of this repository into yours, and keep linked for future upgrades.
```
Comment (davbrito) [inline README.md:23]: This is wrong!!!

CppRex is a header only library. As so it doesn't need to be built before usage.

**Review by davbrito — COMMENTED** [review]
:eyes:

File: README.md (hunk 1 above)
Comment (alfosuag) [reply README.md:23]: Really? Great, then. I will fix that.

**Comment by davbrito** [comment]
Please, rebase this branch into `master`. I made some changes in the repo.

**Comment by alfosuag** [comment]
How should I do the rebase?

> Please, rebase this branch into `master`. I made some changes in the repo.

@davbrito

**Comment by alfosuag** [comment]
I have rebased the branch. I'm ready for review again! @davbrito

File: README.md (hunk 1 above)
Comment (alfosuag) [reply README.md:23]: This was my redaction:

```suggestion
If you want to link this library to a project for its usage, at the time, we recommend putting the library source code at `./deps/cpprex` relative path of your application project directory. Make use of [Git submodules](https://gist.github.com/gitaarik/8735255) to integrate this repository into yours, but then, while keeping linked it for future upgrades.
```

I did some other changes about improving my English grammar in the other text.

**Pull request closed without merging**